saleor/graphql/account/resolvers.py

```diff
@@ -0,0 +1,25 @@
+from graphql_jwt.decorators import login_required
+
+from ...account import models
+from ..utils import get_node
+from .types import User
+
+
+@login_required
+def resolve_users(info):
+    user = info.context.user
+    if user.get_all_permissions() & {'account.view_user', 'account.edit_user'}:
+        qs =  models.User.objects.all().prefetch_related('addresses')
+        return qs
+    # FIXME: Returning 'None' makes graphene return all users
```

Comment:
This is unfortunately expected. Returning `None` effectively says "I don't know how to resolve this" as internally there is no distinction between cases when the resolver returns `None` and when there is no resolver method defined.